packages/ui/src/components/logo.tsx

```diff
@@ -65,7 +65,7 @@ const Logo: FC<LogoProps> = ({ name, size = 24, original = true, ...props }) =>
     <svg
       width={size}
       height={size}
-      viewBox="0 0 24 24"
+      viewBox={`0 0 ${size} ${size}`}
```

Comment:
Why do we want to change the view box like this? This doesn't change the size of the logo, on the vector canvas it's drawn on. Changing the view box without changing the vectors that are being drawn can cause the logo to be drawn small up in the top left corner (when the view box is larger than the expected size) or render outside of visible area (when the view box is smaller than the expected size). There is no connection between `viewBox` and `width`/`height` in SVG

Reply:
This is what happens when `size` is set to `48` <img width="313" alt="Screenshot 2025-05-21 at 12 33 12 (2)" src="https://github.com/user-attachments/assets/43268c4a-ffdf-4f6a-91e7-8e9bed251df4" />

Reply:
And with `12`
<img width="209" alt="Screenshot 2025-05-21 at 12 35 30 (2)" src="https://github.com/user-attachments/assets/d11382c3-5530-4f7d-87c0-39b09a5fd63c" />